lib/teams.rb

```diff
@@ -0,0 +1,18 @@
+class Team
```

Comment:
The file name is teams and the class is Team. I suggest we decide which to use and make those changes before merging.

Reply:
ah - so looks like I created the teams.rb and you had created the team.rb which got merged into master (this explains why all the tests passed for team_test - the classes are really similar). I can't update this to team.rb since this file already exists. Seems like our options are for me to close this PR, correct on my local repo, then open a new PR or we can merge with the typo, correct, and then submit another PR.

Reply:
Deleted teams.rb class file from PR. Adding separate PR for team.rb update.